### PR TITLE
FOGL-236 Make the TI Sensor Tag 8 different assets to resolve clashes

### PR DESCRIPTION
**Pull request by MarkRiddoch**
for x, y, and z and also to improve upload to PI - 0 default readings
issue.

Bring mouse and Wall clock into line so these may be used to feed
load into PI

### CODE REVIEW

File: examples/node-red/multi-asset.json [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+                "7d968a2c.9214c4"
+            ]
+        ]
+    },
+    {
+        "id": "7d968a2c.9214c4",
+        "type": "function",
+        "z": "63b6fb03.fef234",
+        "name": "MakeJSON",
+        "func": "msg.payload = \n{\n    readings : {\n        button : 'Down'\n    },\n    timestamp : new Date(msg.payload).toISOString(),\n    asset: 'mouse'\n}\n;\nreturn msg;",
```
Comment (sstefano69) [inline examples/node-red/multi-asset.json]: readings -> sensor_values

File: examples/node-red/multi-asset.json [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+                "6b562777.6b6c78"
+            ]
+        ]
+    },
+    {
+        "id": "6b562777.6b6c78",
+        "type": "function",
+        "z": "63b6fb03.fef234",
+        "name": "MakeJSON",
+        "func": "msg.payload = {\n    timestamp : new Date().toISOString(),\n    asset : 'TI ' + msg.topic,\n    readings : msg.payload\n};\nreturn msg;",
```
Comment (sstefano69) [inline examples/node-red/multi-asset.json]: readings -> sensor_values

File: examples/node-red/multi-asset.json [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+                "b0c80e36.b2b7b"
+            ]
+        ]
+    },
+    {
+        "id": "b0c80e36.b2b7b",
+        "type": "function",
+        "z": "63b6fb03.fef234",
+        "name": "MakeJSON",
+        "func": "msg.payload = \n{\n    readings : {\n        tick : \"tock\"\n    },\n    timestamp: new Date(msg.payload).toISOString(),\n    asset: 'wall clock'\n};\nreturn msg;",
```
Comment (sstefano69) [inline examples/node-red/multi-asset.json]: readings -> sensor_values

**Comment by MarkRiddoch** [comment]
Updated readings to sensor_values

**Comment by sstefano69** [comment]
ok

**Pull request closed without merging**